java/overrides.js

```diff
@@ -60,7 +60,8 @@
     "appium android landscape mode check region on android 7": {skipEmit: true},
     "appium android landscape mode check window on android 10": {skipEmit: true},
     "appium android landscape mode check window on android 7": {skipEmit: true},
-    "should work with beforeCaptureScreenshot hook": {skip: true}
+    "should work with beforeCaptureScreenshot hook": {skip: true }, 
+    'should get dom pseudo elements when check window': {skipEmit: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9515763)
```suggestion
    "should get dom pseudo elements when check window": {skipEmit: true},
```